core/certificate.go

```diff
@@ -16,18 +16,29 @@
 
 package core
 
+import (
+	"crypto"
+)
+
+type NodeMeta interface {
```

Comment:
BaseNode maybe?

Reply:
I've already changed from NetworkNode to NodeMeta (Dmitry asked me for this)